src/Physics/Multinucleon/EventGen/MECGenerator.cxx

```diff
@@ -628,6 +628,13 @@ void MECGenerator::SelectNSVLeptonKinematics (GHepRecord * event) const
   // This way if someone reuses this code, they are not tripped up by it.
   TMax = Enu - LepMass;
 
+  // Warn if fQ3Max value is suspect: 
+  if(fQ3Max>1.21){
```

Comment:
This was just because it was so easy for users to forget to change fQ3Max when swapping SuSA and Valencia MEC in a custom configuration. I added a comment to make this clear.